### ISSUE TITLE
Conv3D for WLASL

### ISSUE THREAD

**Issue by EvgeniaChroni** [jackyjsy/CVPR21Chal-SLR#14]
Hello thank you so much for your code. 

I am trying to train Conv3D for WLASL dataset but the accuracy is too low. 
Do you have any idea why this is happening ?

Thank you in advance.

**Comment by jackyjsy** [comment]
Hi there, thanks for reaching out. Can you elaborate more details of your training settings (e.g. learning rate, pretrained model, epochs, and batchsize)? What's the accuracy you got?

**Comment by EvgeniaChroni** [comment]
Yeah sure , 
I use Sign_Isolated_Conv3D_clip.py to run the code. 
num_classes = 2000 
epochs = 100
batch_size = 32
learning_rate = 1e-3

I also use the pretrained model that you provide but I train only the last layer (model.fc1 = nn.Linear(model.fc1.in_features, num_classes)).

Thank you again

**Comment by EvgeniaChroni** [comment]
Ok thank you for the info.

I also used gen_frames.py for data preprocessing and same sample_duration=32 

Are you planning to provide code also for WLASL?

**Comment by jackyjsy** [comment]
I did the same thing for data preprocessing as well. 
Yes, we are preparing a journal extension of this workshop paper. 
A preprint version with code will be released shortly (targeting this month).
